lib/trento/domain/sap_system/commands/mark_application_instance_absent.ex

```diff
@@ -0,0 +1,15 @@
+defmodule Trento.Domain.Commands.MarkApplicationInstanceAbsent do
+  @moduledoc """
+  Mark an application instance as absent from the monitoring system.
```

Comment:
I don't really like this `from the monitoring system`. It is kind of implicit, isn't it?